src/adr/0040-quality-levels-definitions.md

```diff
@@ -0,0 +1,51 @@
+# 40. Quality levels definition
+
+Date: 2020-06-21
+
+## Status
+
+Accepted (lazy consensus)
+
+## Context
+
+James as a project delivers several artifacts, and features. In order for project users to better
```

Comment:
What do you mean by artifacts here?

Reply:
`artifacts` => things James project users consumes.

 - Libraries as stated here: 
 - Mail Servers
 - Command line tools
 - Jars that can be used as an extension for a running James server